celestia/peers.txt

```diff
@@ -4,4 +4,5 @@ f5bce79feec9068154e7f50fe8db7a8191a58663@82.220.38.204:10056
 629c36a551ee1a3c8bb3ecee6e00a9977a60e53e@65.108.46.248:56656
 d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656
 a41b4bc451b1b71d537aa1018226e08d7fa7e44e@5.255.77.44:26656
-2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756
\ No newline at end of file
+2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`